.github/workflows/ci.yaml

```diff
@@ -62,32 +79,13 @@ jobs:
         with:
           path: artifact
 
-  brittany:
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@v2
-      - run: echo '::add-matcher::config/brittany-problem-matcher.json'
-      - run: git ls-files '*.hs' | xargs docker run --volume "$PWD:$PWD" --workdir "$PWD" itprotv/brittany:64830d767a4eb58aff4e1f1d72fcb489be367893 brittany --check-mode --config-file config/brittany.yaml
-
-  hlint:
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@v2
-      - run: curl --location --output hlint.tar.gz https://github.com/ndmitchell/hlint/releases/download/v3.2.7/hlint-3.2.7-x86_64-linux.tar.gz
-      - run: gzip --decompress hlint.tar.gz
-      - run: tar --extract --file hlint.tar
-      - run: echo '::add-matcher::config/hlint-problem-matcher.json'
-      - run: hlint-3.2.7/hlint --hint config/hlint.yaml src
-
   release:
     needs: build
     if: github.event_name == 'release'
     runs-on: ubuntu-20.04
     steps:
 
       - uses: actions/download-artifact@v2
-        with:
-          name: artifact
```

Comment:
Ugh, this is so annoying. If you don't specify a name, this action will download all artifacts and put them in directories based on their name. If you do specify a name, it will download that entire artifact into the current working directory.